online-shop/src/components/listProducts/listProducts.styles.css

```diff
@@ -0,0 +1,8 @@
+.listItems{
+    display: flex;
+    flex-direction: row;
+    margin-top:100px;
+    justify-content: center;
+    border:1px solid;
+
```

Comment:
Don't leave empty lines in the styling classes

Reply:
Here you will have to use JSS in order to create a styling object. (`makeStyles((theme: Theme) => createStyles({...}))`)